docs/shared-services-cluster/08_5_minio_ssc.md

```diff
@@ -0,0 +1,46 @@
+# Install Minio
+
+We will deploy Minio as a target for harbor images and backups.
```

Comment:
Shouldn't we say `as a target for harbor images and Velero backups.` ?

Reply:
good suggestion.  I'll make the update.